src/mds/MDSRank.cc

```diff
@@ -3183,7 +3183,7 @@ void MDSRank::command_dump_tree(const cmdmap_t &cmdmap, std::ostream &ss, Format
   std::lock_guard l(mds_lock);
   CInode *in = mdcache->cache_traverse(filepath(root.c_str()));
   if (!in) {
-    ss << "root inode is not in cache";
+    ss << "inode for path '" << filepath(root.c_str()) << "' is not in cache";
```

Comment:
when we say we are traversing through mdcache, the "root inode" is self explanatory that it is `/`.

Reply:
Hey @dparmar18, see the original description for https://tracker.ceph.com/issues/57059. Perhaps there is still a way to make this message more informative?

Reply:
> when we say we are traversing through mdcache, the "root inode" is self explanatory that it is `/`.

IMO the `root` here means the root of a path passed from the `dump tree` command, such as a path `a/b/c/d/`, the `root` here should be `a/`.

Reply:
hmm so this is quite misleading, say for e.g. `ceph tell mds.0 dump tree foo` tells `root inode is not in cache` which definitely is misleading and needs to be fixed. My bad i didn't take a look at the tracker, thanks @ljflores @lxbsz